README.md

```diff
@@ -18,6 +18,7 @@ Features:
 ## Have a bug report or a feature request?
 You can see our templates by visiting our [issue center](https://github.com/CypherpunkArmory/UserLAnd/issues).
 
+You can also chat with us on [slack](https://join.slack.com/t/userlandtech/shared_invite/enQtNTIyNzM4OTgwNjU3LWNhOWEzNmZiZTFiMGU1Zjg1MThhNTY0M2IwODY2OWQ5N2U1YzA2ZGUwMDVkOGVlZjBhNDYzMjgzYWZlZTBmZGU)
```

Comment:
This is a link from slack. I saw on our website we link to https://communityinviter.com/apps/userlandtech/userland

Im not sure the better choice

Reply:
Either seems fine to me.